docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
             - 6379:6379
         volumes:
             - ./config/redis.conf:/redis.conf
-        command: [ "redis-server", "/redis.conf" ]
+        command: [ "redis-server", "/redis.conf", "--port", "6379" ]
```

Comment:
Czemu to służy?

Reply:
To jest w zasadzie zmiana Adriana, ale z tego co wiem to redis domyślnie się uruchamia na porcie 6379, więc być może jest to do czegoś potrzebne

Reply:
tak trzeba ustawić różne porty dla redisów, próbowałem inaczej, nie dało rady; bez tego wywala się komunikacja z serwerem